packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -1,46 +1,62 @@
-import colorsV2 from '@celo/react-components/styles/colors.v2'
-import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import HamburgerIcon from '@celo/react-components/icons/Hamburger'
+import colors from '@celo/react-components/styles/colors.v2'
 import { useNavigation } from '@react-navigation/native'
 import * as React from 'react'
 import { StyleSheet, TouchableOpacity, View } from 'react-native'
-import Logo from 'src/icons/Logo.v2'
-import Hamburger from 'src/navigator/Hamburger'
+import Animated from 'react-native-reanimated'
 
-const DrawerTopBar = () => {
+interface Props {
+  middleElement?: React.ReactNode
+  scrollPosition?: Animated.Value<number>
+}
+
+// Test mocks would not work if import these together with Animated
+const { cond, greaterThan } = Animated
```

Comment:
Oh ok, I believe this is because we need to do this in `__mocks__/react-native-reanimated`:
```
module.exports = require('react-native-reanimated/mock')
```

Reply:
Good catch, did not think of that. So used to `import/export` nowadays 😄